test/BaselineOutput/Common/AveragedPerceptron/netcoreapp/AveragedPerceptron-CV-breast-cancer.PAVcalibration.txt

```diff
@@ -1,58 +1,58 @@
 Instance	Label	Score	Probability	Log-loss	Assigned
 5	1	11.285979	1	-0	1
-6	0	-0.9347124	0.09090909	0.13750352804950119	0
+6	0	-0.93471193	0.09090909	0.13750352804950119	0
```

Comment:
Are these changes improvements, regressions, something else?

Reply:
The biggest thing is just that the dataset we are using for these tests is tiny, so any minor change we make to anything will affect the accuracy. The tests I ran on actual full size datasets didn't really show any changes in accuracy.